compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -5033,6 +5039,19 @@ public void visit(BLangXMLElementAccess xmlElementAccess) {
         result = rewriteExpr(invocationNode);
     }
 
+    private BLangInvocation generateRequireTypeLangLibFunction(BLangAccessExpression fieldAccessExpr) {
+        ArrayList<BLangExpression> requiredArgs = new ArrayList<>();
+        BType typedescType = new BTypedescType(fieldAccessExpr.type, null);
+        BLangTypedescExpr typedescExpr = new BLangTypedescExpr();
+        typedescExpr.resolvedType = fieldAccessExpr.type;
+        typedescExpr.type = typedescType;
+        requiredArgs.add(typedescExpr);
+        fieldAccessExpr.desugared = true;
+        BLangInvocation invocationNode = createLangLibInvocationNode(VALUE_REQUIRE_TYPE, fieldAccessExpr,
```

Comment:
Local variable `invocationNode` is redundant and we can return directly

Reply:
Removed the `invocationNode` variable.